src/frontend/Komponenter/Migrering/MigrerBarnetilsyn.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useMemo, useState } from 'react';
+import { useApp } from '../../App/context/AppContext';
+import { byggHenterRessurs, byggTomRessurs, Ressurs, RessursStatus } from '../../App/typer/ressurs';
+import { AxiosRequestConfig } from 'axios';
+import { useToggles } from '../../App/context/TogglesContext';
+import { ToggleName } from '../../App/context/toggles';
+import { IFagsakPerson } from '../../App/typer/fagsak';
+import styled from 'styled-components';
+import { Button } from '@navikt/ds-react';
+import { visMigrertStatus } from './MigrerFagsak';
+
+const StyledKnapp = styled(Button)`
+    margin: 0.25rem;
+`;
+
+const MigrerBarnetilsyn: React.FC<{
+    fagsakPerson: IFagsakPerson;
+}> = ({ fagsakPerson }) => {
+    const { axiosRequest } = useApp();
+    const { toggles } = useToggles();
+    const [migrertStatus, settMigrertStatus] = useState<Ressurs<string>>(byggTomRessurs());
+
+    const { id: fagsakPersonId } = fagsakPerson;
+
+    const migrerFagsakConfig: AxiosRequestConfig = useMemo(
+        () => ({
+            method: 'POST',
+            url: `/familie-ef-sak/api/migrering/${fagsakPersonId}/barnetilsyn`,
+        }),
+        [fagsakPersonId]
+    );
+
+    if (!toggles[ToggleName.kanMigrereBarnetilsyn]) {
+        return null;
+    }
+
+    const migrerFagsak = () => {
+        settMigrertStatus(byggHenterRessurs());
+        axiosRequest<string, void>(migrerFagsakConfig).then((res: Ressurs<string>) => {
+            settMigrertStatus(res);
+        });
+    };
+
+    return (
```

Comment:
Vises denne for alle fagsaker nå? Klarer ikke å skjønne om det bare er gyldig for de som har sak i infotrygd og ikke allerede har sak i ny løsning 🤔

Reply:
Den vises på infotrygdsiden, uavhengig om man har fagsak eller ikke, likt overgangsstønad gjøres.
Det burde kanskje være en sjekk på om det finnes behandlinger på fagsaken, men det finnes ikke idag :)